app/src/components/v-select/v-select.vue

```diff
@@ -186,6 +191,10 @@ export default defineComponent({
 			type: Boolean,
 			default: true,
 		},
+		groupsClickable: {
+			type: Boolean,
+			default: false,
+		},
```

Comment:
This might have to be on a per-group level instead of as a general prop on the whole v-select